module/cli_controller.py

```diff
@@ -129,7 +98,7 @@ def process(self) -> None:
 
         self.check_input()
 
-    def stop(self) -> None:
+    def stop(self, params: list = None) -> None:
```

Comment:
because stop is a global command, and we give parameters to all global command calls, this enables more flexibility when adding other commands without writing exceptions for every single one of them

Reply:
Ah didn't see